.github/workflows/main.yml

```diff
@@ -75,44 +92,68 @@ jobs:
         uses: actions/checkout@v2.3.4
 
       - name: Install apt packages
-        run: sudo apt-get update && sudo apt-get install -y texlive-latex-recommended texlive-xetex gettext nginx
+        run: sudo apt-get update &&
+             sudo apt-get install -qqy texlive-latex-recommended texlive-xetex gettext nginx
         env:
           DEBIAN_FRONTEND: noninteractive
 
+      - name: Cache cpan modules
+        uses: actions/cache@v2.1.3
+        with:
+          # A list of files, directories, and wildcard patterns to cache and restore
+          path: $PERL5LIB
+          # An explicit key for restoring and saving the cache
+          key: ${{ runner.os }}-cpan-${{ hashFiles('**/cpanfile') }}
+          # An ordered list of keys to use for restoring the cache if no cache hit occurred for key
+          restore-keys: ${{ runner.os }}-cpan-
+
       - name: install with cpm
         run: |
           cpanm --notest App::cpm
           cpm install -g -v --resolver=metacpan --notest --with-develop --feature=starman --feature=latex-pdf-ps --feature=openoffice --feature=xls --feature=edi
 
+      - name: Patch Weasel
+        run: |
+          sudo cp -r utils/test/Weasel /opt/hostedtoolcache/perl/${{ matrix.perl }}/x64/lib/site_perl/${{ matrix.perl }}
+
       - name: Build JS
         run: make dojo
 
       - name: Other preparations
         run: |
-          mkdir screens logs
+          mkdir -p screens logs
           cp doc/conf/ledgersmb.conf.default ledgersmb.conf
 
       - name: Starting 'starman'
         run: starman --preload-app --pid starman.pid --workers 2 -Ilib -Iold/lib --port 5762 bin/ledgersmb-server.psgi &
+        # Use plackup to add reloading capability & debug
```

Comment:
This comment seems way out of place? Surely our CI machines don't need reloading capability?